src/lib/components/chat/ChatMessages.svelte

```diff
@@ -18,6 +18,7 @@
 	export let currentModel: Model;
 	export let settings: LayoutData["settings"];
 	export let models: Model[];
+	export let preprompt: string | undefined;
```

Comment:
systemPrompt instead of preprompt?

Reply:
Looking at it, we've been calling it `preprompt` in the .env for [a while now](https://github.com/huggingface/chat-ui/commit/cd6894da7d7940315f2e69bbbe670dc1d4ecb575#diff-9a6d1a5a34d34f4a64bbe39b0cbf3d6a6e6cbd98943f8503a6aeecc091829d25), and if we change it, it means a breaking change for users, as they will need to update their .env. 

If you think it's worth the change I can still do it though